src/services/crypto.service.ts

```diff
@@ -452,6 +452,17 @@ export class CryptoService implements CryptoServiceAbstraction {
         return b64DecValue;
     }
 
+    async sha1(password: string): Promise<string> {
+        const hash = await Crypto.subtle.digest(
+            {
+                name: 'SHA-1',
+            },
+            UtilsService.fromB64ToArray(window.btoa(password)),
```

Comment:
Why not the utf-8 to array util?

Reply:
Whops, nice catch. Was experimenting why the result from `fromBufferToUtf8` threw errors. (Needed to convert it to Hex, not Utf8)